junit4-engine/src/main/java/org/junit/gen5/engine/junit4/descriptor/JUnit4TestDescriptor.java

```diff
@@ -89,15 +87,8 @@ public boolean isContainer() {
 		Set<TestTag> result = new LinkedHashSet<>();
 		getParent().ifPresent(parent -> result.addAll(parent.getTags()));
 		// @formatter:off
-		getDeclaredCategories().ifPresent(categoryClasses ->
-			stream(categoryClasses)
-				.map(ReflectionUtils::getAllAssignmentCompatibleClasses)
-				.flatMap(Collection::stream)
-				.distinct()
-				.map(Class::getName)
-				.map(TestTag::new)
-				.forEachOrdered(result::add)
-		);
+		getDeclaredCategories().ifPresent(categoryClasses -> stream(categoryClasses).map(ReflectionUtils::getAllAssignmentCompatibleClasses)
+		        .flatMap(Collection::stream).distinct().map(Class::getName).map(TestTag::new).forEachOrdered(result::add));
```

Comment:
I'm assuming that was Stefan's IDE.

But I agree: let's undo this reformatting.